### DISCUSSION TITLE
Private CouchDB docker - any reasons to enable encryption?

### DISCUSSION THREAD

**Discussion by techn0pia** [vrtmrz/obsidian-livesync#334] [Q&A]
Hi all,

I want to run Obsidian LiveSync with a local docker installation of CouchDB protected via Caddy/Letsencrypt.

As the connection is via HTTPS, and I can only connect to it inside my HomeNetwork via VPN - is there any reason to enable encryption?
My assumption is, without encryption it's more stable because less complexity, and also less energy consumption on my mobile devices.

Your experience/opinion? Thank you :)

**Comment by vrtmrz** [comment]
Thank you for asking me! This is an important question.

Since SSL has been enabled, our communication traffics might be safe, with some minor exception. If the remote server is encrypted automatically, it could be also safe.

However, if the storage of the remote server is not encrypted, it means we have some vulnerabilities.
It may be censorship of the company or something. It may be an exposure of pending intellectual property through security scanning.
(CouchDB stores data rather naively).
Even if we are not suspected, the possibility cannot be dismissed that the above incidents may occur due to the physical arrest of our neighbours or analysis by detection of annexed servers.

These are the reasons why I have enabled that in default or recommendation.

Of course, we can disable encryption if we can rule out that risk.

Thank you for the opportunity to explain!
